src/QMCHamiltonians/SOECPotential.h

```diff
@@ -107,6 +111,9 @@ class SOECPotential : public OperatorBase
   //multi walker resource
   ResourceHandle<SOECPotentialMultiWalkerResource> mw_res_handle_;
 
+  //flag to use fast evaluation
+  bool use_fast_evaluation_;
```

Comment:
If we set its value via constructor, please add const.

Reply:
see above